clang/test/dpct/cmake_migration/case_048/out/input.cmake

```diff
@@ -0,0 +1 @@
+add_library(target foo.cpp)
```

Comment:
This change is not related to the current PR; pls submit a separate PR for this change.